app/resources/documentation/manual/manual.md

```diff
@@ -1,20 +1,20 @@
-# TrenchBroom help documentation
+# TrenchBroom manual documentation
 
 ## The Build Process
 
 TrenchBroom's documentation is contained in a single markdown file (index.md). This file is converted into HTML using [pandoc](http://www.pandoc.org) during the build process. The build process also converts our custom macros (see below) into javascript snippets that output some information into the help document.
 
 If you want to preview the generated HTML without doing a full build, you can just build the GenerateHelp target. Change into your build directory and run
 
-    cmake --build . --target GenerateHelp
+    cmake --build . --target GenerateManual
 
-You will find the generated documentation files in a directory called "gen-help". If you add new resources such as images to the help files, you have to refresh your cmake cache first by running
+You will find the generated documentation files in a directory called "gen-manual" (/build/app/gen-manual). If you add new resources such as images to the manual's files, you have to refresh your cmake cache first by running
```

Comment:
We should not assume "build" as the build directory. Change this to something like "<build-directory>/app/gen-manual"). Maybe it would be better to use inline code markup to highlight paths and names.

Reply:
It should still say that it's in the build dir though, otherwise these instructions are not helpful ;)
how about `{build directory}/app/gen-manual`

Reply:
Haha, it swallowed my `<build dir>` as markup ;-)

Reply:
Alright, is (`<build dir>/app/gen-manual`) what you're looking for then?